bundles/org.openhab.binding.ecotouch/src/main/resources/OH-INF/i18n/ecotouch_de.properties

```
@@ -0,0 +1,249 @@
+# binding
+binding.ecotouch.name = EcoTouch Binding
+binding.ecotouch.description = �berwachung und Steuerung von Waterkotte EcoTouch W�rmepumpen
+
+# thing types (geo)
+thing-type.ecotouch.geo.label = Waterkotte EcoTouch Geo
+thing-type.ecotouch.geo.description = Waterkotte EcoTouch Geo Erdw�rmepumpe
+
+# thing types (air)
+thing-type.ecotouch.air.label = Waterkotte EcoTouch Air
+thing-type.ecotouch.air.description = Waterkotte EcoTouch Air Luftw�rmepumpe
+
+# thing type config description (geo)
+thing-type.config.ecotouch.geo.ip.label = IP Adresse / Hostname der W�rmepumpe
```

Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Please check all.

```suggestion
thing-type.config.ecotouch.geo.ip.label = IP Adresse / Hostname
```